plugins/hls-explicit-imports-plugin/src/Ide/Plugin/ExplicitImports.hs

```diff
@@ -444,6 +479,10 @@ abbreviateImportTitle input =
           else actualPrefix <> suffixText
   in title
 
+squashedAbbreviateImportTitle :: T.Text -> T.Text
```

Comment:
How is this different to what we do on line 170? Why are we doing different things?

Reply:
I think I worked it out: the previous thing has the whole import spec in it. I think it would be better to just create the two bits of the import spec separately when we generate the title, then we can use only the pieces we need.

We'd then need to tweak `abbreviateImportTitle` to just abbreviate the list of imported items, and then we can glue it onto the previous part of the import or not depending on whether we're creating an inlay hint or a code lens.

Reply:
Not only that, `abbreviateImportTitle` will retain a lot of indentation (shown as spaces after cutting by line), and inlay hints label will need remove those spaces because they are displayed at the end of each imports statement to avoid being too long.

Reply:
The thing I was suggesting is that in the `ImportActions` rule we could compute both a) the prefix of the import (`import Foo`) and the suffix (`(bar, baz)`). Then for the code lenses we can concatenate them together and abbreviate, and for the lens we can just use the suffix?